challenge_14/java/jdfurlan/src/ReversingALinkedList.java

```diff
@@ -0,0 +1,71 @@
+
+import java.util.Scanner;
+
+public class ReversingALinkedList {
+
+	private static void printList(ListNode head) {
+		while (head != null) {
+			System.out.print(head.data);
```

Comment:
Ya the challenge states "The strings in standard input will be space separated." I have to assume expected input.